packages/fast-jss-manager-react/README.md

```diff
@@ -1,2 +1,67 @@
+# React JSS Manager
+`fast-react-jss-manager` is a React higher order component (HOC) library for managing component JSS stylesheets. It facilitates using JSS with your react components by controlling when to add, update, and remove a JSS stylesheets. It also allows top-down variable injection into JSS stylesheet functions using the exported `DesignSystemProvider`.
 
-A React higher order component (HOC) library for managing component JSS stylesheets.
+
+## Usage
+### Create a react component
+`manageJss` is a HOC that passes as props the generated classes from a given stylesheet. The generated classes are made available on the `managedClasses` prop of the rendered component.
+```jsx
+// button.js
+import * as React from "react";
+
+class Button extends React.Component {
+    render(): {
+        return (
+            <button className={this.props.managedClasses.button}>
+                {this.props.children}
+            </button>
+        );
+    }
+}
+```
+
+### Create a JSS stylesheet
+You should also create a JSS stylesheet - for detailed information please reference (https://github.com/cssinjs/jss)[https://github.com/cssinjs/jss]. The JSS instance used by the manager uses (jss-preset-default)[https://github.com/cssinjs/jss-preset-default] so feel free to use any syntaxes supported by those plugins.
```

Comment:
actually (for the cssinjs link) you don't need to re-state the link, in the () you can put some kind of nice text instead of a link, or just leave it off altogether and it should render the link with the address

Reply:
resolved